source/elements/oneMKL/source/architecture/other_architecture.inc.rst

```diff
@@ -11,10 +11,33 @@ This section covers all other features in the design of oneMKL architecture.
 
 .. _onemkl_spec_current_version:
 
-Current Version of this oneMKL Specification
+Version of oneMKL Specification
 +++++++++++++++++++++++++++++++++++++++++++++
 
-This is the oneMKL specification which is part of the oneAPI specification version 1.0.0.
+This is the oneMKL specification which is part of the oneAPI specification.
+
+The implementation must define a macro, `ONEMKL_SPEC_VERSION` as having the numerical value of the specification version.
+
+The specification version can be created by appending all digits of the specification version in the format of <MAJOR><MINOR>. MINOR version always uses two digits. This version can be used to check the compatibility of the implementation with the specification version.
+
```

Comment:
I have a concern that over time we may reach a MAJOR version larger than 9, what will happen then?  should we always use 4 digits ?  0101,  0102, etc ?

Reply:
actually since we are using a macro number through `#define`, I suppose it really doesn't matter, does it ...

Reply:
why are we ignoring the revision ?

Reply:
For the major version, yes, we will keep enabling additional digits as we move beyond `9` or `99`, because it won't affect the numerical comparison. Having a `0` prefix for single digit major versions will need us to update the spec if we ever approach `100`. I did not see a strong reason to do this, so I kept it simple. It is still an option if we feel it will add some advantage.

We are ignoring the revision because as per https://github.com/uxlfoundation/oneAPI-spec/pull/561#discussion_r1725767506, the revision is used only to update the documentation, not the implementation.